labelbox/schema/dataset.py

```diff
@@ -54,13 +56,18 @@ def create_data_row(self, **kwargs):
         return self.client._create(DataRow, kwargs)
 
     def create_data_rows(self, items):
-        """ Creates multiple DataRow objects based on the given items.
+        """ Creates multiple DataRow objects based on the given `items`.
+        
         Each element in `items` can be either a `str` or a `dict`. If
-        it's a `str`, then it's interpreted as a file path. The file
+        it is a `str`, then it is interpreted as a local file path. The file
         is uploaded to Labelbox and a DataRow referencing it is created.
         If an item is a `dict`, then it should map `DataRow` fields (or their
-        names) to values. At the minimum it must contain a `DataRow.row_data`
-        key and value.
+        names) to values. At the minimum an `item` passed as a `dict` must 
+        contain a `DataRow.row_data` key and value.
+
+        >>> dataset.create_data_rows([
+                {DataRow.row_data:"http://my_site.com/photos/img_01.jpg"},
+                "path/to/file2.jpg"])
```

Comment:
Please change to:
```
>>> dataset.create_data_rows([
>>>     {DataRow.row_data:"http://my_site.com/photos/img_01.jpg"},
>>>     "path/to/file2.jpg"
>>> ])
```
Note the leading `>>>` in every line, and I just moved the final `])` to new line for even better readibility.